modules/tensor_mechanics/doc/content/documentation/systems/BCs/Pressure/index.md

```diff
@@ -1,6 +1,40 @@
 # Pressure System
 
-The Pressure Action is used to create a set of pressure boundary conditions for a string of displacement variables; the typical use case for this action is the application of hydrostatic pressure.
+!syntax description /BCs/Pressure/PressureAction
```

Comment:
I think I like the way you have it set up better now. I'm curious how you like it. One thing that's weird is that the path to the Pressure system page and the regular Pressure BC looks identical in the breadcrumb list at the top of the page.